src/main/java/meteordevelopment/meteorclient/systems/modules/render/BetterTooltips.java

```diff
@@ -80,7 +80,7 @@ public class BetterTooltips extends Module {
     );
 
     public final Setting<Boolean> alwaysShow = sgGeneral.add(new BoolSetting.Builder()
-        .name("always-show")
+        .name("bypass-'hideflag'-nbt")
```

Comment:
I think the old name is better, a normal user doesn't know what that means and in this case the description won't help either.

Reply:
cap, the name "always show" is so ambiguous